.dockerignore

```diff
@@ -0,0 +1,15 @@
+__pycache__
+*.pyc
+*.pyo
+*.pyd
+*.db
+*.env
+README.md
+Dockerfile
+.venv
+result.png
+checklist.md
+.gitignore
+.github
```

Comment:
The .github directory is usually not needed in the Docker image unless it contains runtime configurations.